docs/standard/native-interop/best-practices.md

```diff
@@ -251,6 +251,56 @@ finally
 }
 ```
 
+## Windows and non-Windows data type considerations
+
+There are types in the C language that have latitude in how they are defined. When writing cross-platform interop, cases can arise where platforms differ and can cause issues if not considered.
+
+### C `long`
+
```

Comment:
```suggestion
C `long` and C# `long` are not the same types. Using C# `long` to interop with C `long` is almost never correct.
```
This is very common newbie mistake. I think it is worth highlighting.

Reply:
BTW in practice does this apply to C++ as well (since many times folks are writing C in code they consider C++)

Reply:
>BTW in practice does this apply to C++ as well (since many times folks are writing C in code they consider C++)

Yep. Exact same behavior.

Reply:
Right I just meant, will folks writing C++ read this and skip over? Perhaps it should say C/C++. Whatver you think.

Reply:
Great suggestion. Will update.